projects/lib/src/admin/components/page-list/page-list.component.ts

```diff
@@ -56,8 +61,11 @@ export class PageListComponent implements OnInit, OnChanges {
   loading = true;
   list: JDocument[];
   selected?: JDocument;
+  sortBy: string;
+  faSortUp = faSortUp;
+  faSortDown = faSortDown;
 
-  constructor(private spinner: NgxSpinnerService) { }
+  constructor(private spinner: NgxSpinnerService, private formBuilder: FormBuilder) { }
```

Comment:
it looks like `formBuilder` is no longer being used in this component, can you remove it please?